drag.lua

```diff
@@ -113,8 +117,11 @@ local function drag_event_handler(e)
 			return
 		end
 		assert(drag_win)
+		if drag_win:isFullScreen() then
```

Comment:
These line seem to repeat elsewhere as well. Maybe it's better to extract them to `win_utils.lua` and call them `unfullscreen_win()` or something?